<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceDataBreaker</class>
 <widget class="QWidget" name="DeviceDataBreaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}
QPushButton
{
	font:14px;
    color: rgb(0,0,0);
	background-color: rgb(0, 255, 0);
	border-top: 2px solid white;	
	border-left: 2px solid white;
}
QLabel
{
color:black;
border:2px  solid white;
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>5</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(173, 15, 242);</string>
   </property>
   <property name="text">
    <string>辅助（460V/230V）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelC1CPR">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>35</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>C1-PCR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelC2CPR">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>65</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>C2-PCR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelC3CPR">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>95</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>C3-PCR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDJCPR">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>125</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DJ-CPR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCVTBM2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>215</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CVT-BM2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCVTBM1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>185</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CVT-BM1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelREC">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>155</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Q-REC</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDJPE1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>365</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DJ-PE1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCVTSM1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>305</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CVT-SM1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCVTMT2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>275</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CVT-MT2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDJPE2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>395</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DJ-PE2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCVTSM2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>335</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CVT-SM2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCVTMT1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>245</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CVT_MT1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDJPH1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DJ-PH1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelQTHVTBM2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Q-TH-VT-BM2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelQTHVTBM1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Q-TH-VT-BM1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDJPE3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DJ-PE3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDJPH2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DJ-PH2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDJPE4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DJ-PE4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDJVTSM2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DJ-VT-SM2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelQTHVTMT1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Q-TH-VT-MT1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelQTHVTMT2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Q-TH-VT-MT2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDJVTSM1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>DJ-VT-SM1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTrain1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TrainLogo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTrain2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TrainLogo-reverse.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTrain3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TrainLogo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTrain4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/TrainLogo-reverse.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTrain1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
color:white;</string>
   </property>
   <property name="text">
    <string>HXD2.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTrain2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
border:0px;</string>
   </property>
   <property name="text">
    <string>HXD2.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTrain3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>440</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
color:white;</string>
   </property>
   <property name="text">
    <string>HXD2.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTrain4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
border:0px;</string>
   </property>
   <property name="text">
    <string>HXD2.0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
